docs/base/rust_plugin.rst

```diff
@@ -0,0 +1,140 @@
+Rust plugin
+=============
+
+The Rust plugin can be used for Rust projects that use the Cargo build system.
+
+Keywords
+--------
+
+This plugin uses the common :ref:`plugin <part-properties-plugin>` keywords as
+well as those for :ref:`sources <part-properties-sources>`.
+
+Additionally, this plugin provides the plugin-specific keywords defined in the
+following sections.
```

Comment:
No need to change it if you prefer the current wording — it just sounded a bit rigid to my ear so I thought I'd suggest this:

```suggestion
In addition to the common :ref:`plugin <part-properties-plugin>` and
:ref:`sources <part-properties-sources>` keywords, this plugin provides the following
plugin-specific keywords:
```